src/renderer/services/midgard/pools.ts

```diff
@@ -145,6 +145,11 @@ const createPoolsService = (
    * Ddata of pending `Pools` from Midgard
    */
   const apiGetPoolsPending$: LiveData<Error, PoolDetails> = apiGetPools$({ status: GetPoolsStatusEnum.Staged })
+  /**
+   * Data of all `Pools` or by given status
+   */
+  const apiGetPoolsByStatus$: (status?: GetPoolsStatusEnum) => LiveData<Error, PoolDetails> = (status) =>
+    apiGetPools$({ status })
```

Comment:
and just to clarify. why do we need this ? it only calls `apiGetPools$` factory with `{status}`. is not it ?

Reply:
If `status = undefined`, data of all pools (pending + availabe) are requested

Reply:
ok

Reply:
[Last commit](https://github.com/thorchain/asgardex-electron/pull/833/commits/4119783bbe59d79085800b8d5e4a1e457fd24353) has improved it ^